src/QsCompiler/DataStructures/ReservedKeywords.fs

```diff
@@ -275,7 +279,10 @@ module AssemblyConstants =
     let PerfDataOutputPath = "PerfDataOutputPath"
     let DocsOutputPath = "DocsOutputPath"
     let DocsPackageId = "DocsPackageId"
+    [<Obsolete("Replaced by IsTargetPackage.")>]
```

Comment:
I understand the intent here is to avoid a breaking change for any compiler consumers who may be using the old `GenerateConcreteIntrinsic` property, but since the support for this property is gone, it means any consumers will compile successfully with a warning and then get different behavior. From https://docs.microsoft.com/en-us/dotnet/api/system.obsoleteattribute?view=net-6.0, it appears properties marked as Obsolete only generate warnings instead of errors. I think we should either keep it like this for the warning but then use that value to set the new `IsTargetPackage` property or just remove it such that consumers will get a compilation failure instead.

Reply:
Ah, you are right - good catch. Since this is an undocumented feature used as part of our internal infrastructure and we already have other breaking changes, I am inclined to just remove it.